gazelle/config.go

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2018 The Bazel Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gazelle
```

Comment:
Disclaimer: I don't know Go at all. 

Is this saying that this code is _in_ the `gazelle` package? Shouldn't it be its own package instead of something that already exists?

Reply:
Go has an interesting way of handling packages. Packages are mostly defined by the path you traverse to get there including the hostname of the VCS provider. This file's package would be `github.com/bazelbuild/rules_sass/gazelle` and you would import it with

```go
import "github.com/bazelbuild/rules_sass/gazelle"
```

The line `package gazelle` mostly functions as a sanity check, but it additionally allows you to

1. Put your tests or examples outside the package boundary so that you don't accidentally manipulate internal state or do something that an caller couldn't do.
1. Create a new binary. By setting `package main` the go compiler will emit a statically linked binary instead.